### ISSUE TITLE
Error PostProcessing three digits episodes

### ISSUE THREAD

**Issue by IDerr** [pymedusa/Medusa#1986]
Error while processing episodes with 3 digits, considering the first one as the season,

![](http://i.imgur.com/OEJrIcv.png)

Thanks a lot

**Comment by duramato** [comment]
Do you have anime enabled? you have to since absolute numbering is only for anime, else 3 digits are consider as sep, season episode.

**Comment by IDerr** [comment]
Yes I have

**Comment by duramato** [comment]
mm, this has to do with guessit right? @ratoaq2 & @medariox 
```
# guessit: 2.1.2.dev0  rebulk: 0.8.3.dev0
? One Piece - 722 [HD_1280x720].mp4
: title: One Piece
  season: 7
  episode: 22
  other: HD
  screen_size: 720p
  container: mp4
  mimetype: video/mp4
  type: episode
  parsing_time: 0.331575155258
```

**Comment by medariox** [comment]
Yes. Guessit doesn't seem to detect the absolute numbering.

**Comment by IDerr** [comment]
Think this pull fix this
https://github.com/SickRage/SickRage/pull/2896

**Comment by medariox** [comment]
Nope, our parser is very different. It's just a guessit issue. Not sure if legacy or our version though.

**Comment by p0psicles** [comment]
That Sr is regarding search. This issue is regarding parsing. Or changing a single result/release into information like episode season quality etc.

**Comment by ratoaq2** [comment]
Fixed by #2055

**Comment by IDerr** [comment]
o/
